bridge/core/dom/events/event_target.cc

```diff
@@ -518,6 +518,7 @@ bool EventTarget::FireEventListeners(Event& event,
     assert(i <= size);
   }
   d->firing_event_iterators->pop_back();
+  context->RunRustFutureTasks();
```

Comment:
Move this to rust native function callback

Reply:
But then we will run the tasks after every listener function. Should we do that?